corona-warn-app/cwa-app-android
# RAT registration via link (EXPOSUREAPP-6313)

RAT QR code registration via link

Registration can now be triggered from the consent screen as well in case the qr code is already present.

How to test:
- Click on a RAT qr code (link) -> CWA should be offered, open up on the consent screen and move forward without having to pass the scan screen
- Register a test (pcr, ra) via scan -> should work as before
- Scan a check-in qr code with camera app-> should still work as before
